This patch adds contional openssl support, required for writting
the mtree format. This has been submitted to the project:

http://code.google.com/p/libarchive/issues/detail?id=22

Use "sh build/autogen.sh" to regen the patch if needed.

--- configure.ac.orig	2009-04-16 19:49:04.000000000 +0200
+++ configure.ac	2009-04-17 05:35:59.312891376 +0200
@@ -234,13 +234,17 @@ if test "x$with_lzma" != "xno"; then
   AC_CHECK_LIB(lzma,lzma_stream_decoder)
 fi
 
-AC_CHECK_HEADERS([openssl/md5.h openssl/ripemd.h openssl/sha.h])
-AC_CHECK_HEADERS([md5.h])
-AC_CHECK_HEADERS([ripemd.h rmd160.h])
-AC_CHECK_HEADERS([sha.h sha1.h sha2.h sha256.h])
-AC_SEARCH_LIBS(SHA1_Init, crypto ssl md)
-AC_CHECK_FUNC(MD5_Init)
-if test "x$ac_cv_func_MD5_Init" != "xyes"; then
+AC_ARG_WITH([openssl],
+  AS_HELP_STRING([--without-openssl], [Don't build support for mtree hashes through openssl]))
+
+if test "x$with_openssl" != "xno"; then
+  AC_CHECK_HEADERS([openssl/md5.h openssl/ripemd.h openssl/sha.h])
+  AC_CHECK_HEADERS([md5.h])
+  AC_CHECK_HEADERS([ripemd.h rmd160.h])
+  AC_CHECK_HEADERS([sha.h sha1.h sha2.h sha256.h])
+  AC_SEARCH_LIBS(SHA1_Init, crypto ssl md)
+  AC_CHECK_FUNC(MD5_Init)
+  if test "x$ac_cv_func_MD5_Init" != "xyes"; then
   AC_CHECK_FUNC(MD5Init,
 	[AC_DEFINE(MD5_Init, MD5Init,
 	    [Define to the initializes MD5 context function.])
@@ -249,12 +253,12 @@ if test "x$ac_cv_func_MD5_Init" != "xyes
 	 AC_DEFINE(MD5_Final, MD5Final,
 	    [Define to the generates final MD5 hash function.])
 	])
-fi
-if test "x$ac_cv_func_MD5_Init" = "xyes" -o "x$ac_cv_func_MD5Init" = "xyes" ; then
+  fi
+  if test "x$ac_cv_func_MD5_Init" = "xyes" -o "x$ac_cv_func_MD5Init" = "xyes" ; then
   AC_DEFINE(HAVE_MD5, 1, [Define to 1 if you have the `MD5' functions.])
-fi
-AC_CHECK_FUNC(RIPEMD160_Init)
-if test "x$ac_cv_func_RIPEMD160_Init" != "xyes"; then
+  fi
+  AC_CHECK_FUNC(RIPEMD160_Init)
+  if test "x$ac_cv_func_RIPEMD160_Init" != "xyes"; then
   AC_CHECK_FUNC(RMD160Init,
 	[AC_DEFINE(RIPEMD160_Init, RMD160Init,
 	    [Define to the initializes RIPEMD160 context function.])
@@ -263,12 +267,12 @@ if test "x$ac_cv_func_RIPEMD160_Init" !=
 	 AC_DEFINE(RIPEMD160_Final, RMD160Final,
 	    [Define to the generates final RIPEMD160 hash function.])
 	])
-fi
-if test "x$ac_cv_func_RIPEMD160_Init" = "xyes" -o "x$ac_cv_func_RMD160Init" = "xyes" ; then
+  fi
+  if test "x$ac_cv_func_RIPEMD160_Init" = "xyes" -o "x$ac_cv_func_RMD160Init" = "xyes" ; then
   AC_DEFINE(HAVE_RMD160, 1, [Define to 1 if you have the `RIPEMD160' functions.])
-fi
-AC_CHECK_FUNC(SHA1_Init)
-if test "x$ac_cv_func_SHA1_Init" != "xyes"; then
+  fi
+  AC_CHECK_FUNC(SHA1_Init)
+  if test "x$ac_cv_func_SHA1_Init" != "xyes"; then
   AC_CHECK_FUNC(SHA1Init,
 	[AC_DEFINE(SHA1_Init, SHA1Init,
 	    [Define to the initializes SHA1 context function.])
@@ -277,22 +281,23 @@ if test "x$ac_cv_func_SHA1_Init" != "xye
 	 AC_DEFINE(SHA1_Final, SHA1Final,
 	    [Define to the generates final SHA1 hash function.])
 	])
-fi
-if test "x$ac_cv_func_SHA1_Init" = "xyes" -o "x$ac_cv_func_SHA1Init" = "xyes" ; then
+  fi
+  if test "x$ac_cv_func_SHA1_Init" = "xyes" -o "x$ac_cv_func_SHA1Init" = "xyes" ; then
   AC_DEFINE(HAVE_SHA1, 1, [Define to 1 if you have the `SHA1' functions.])
-fi
-AC_CHECK_FUNC(SHA256_Init)
-if test "x$ac_cv_func_SHA256_Init" = "xyes" ; then
+  fi
+  AC_CHECK_FUNC(SHA256_Init)
+  if test "x$ac_cv_func_SHA256_Init" = "xyes" ; then
   AC_DEFINE(HAVE_SHA256, 1, [Define to 1 if you have the `SHA256' functions.])
-fi
-AC_CHECK_FUNC(SHA384_Init)
-if test "x$ac_cv_func_SHA384_Init" = "xyes" ; then
+  fi
+  AC_CHECK_FUNC(SHA384_Init)
+  if test "x$ac_cv_func_SHA384_Init" = "xyes" ; then
   AC_DEFINE(HAVE_SHA384, 1, [Define to 1 if you have the `SHA384' functions.])
-fi
-AC_CHECK_FUNC(SHA512_Init)
-if test "x$ac_cv_func_SHA512_Init" = "xyes" ; then
+  fi
+  AC_CHECK_FUNC(SHA512_Init)
+  if test "x$ac_cv_func_SHA512_Init" = "xyes" ; then
   AC_DEFINE(HAVE_SHA512, 1, [Define to 1 if you have the `SHA512' functions.])
-fi
+  fi
+fi # with_openssl
 
 # TODO: Give the user the option of using a pre-existing system
 # libarchive.  This will define HAVE_LIBARCHIVE which will cause

--- configure.orig	2009-04-17 05:48:16.335933743 +0200
+++ configure	2009-04-17 05:48:58.063825315 +0200
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.62 for libarchive 2.7.0.
+# Generated by GNU Autoconf 2.63 for libarchive 2.7.0.
 #
 # Report bugs to <kientzle@freebsd.org>.
 #
@@ -791,136 +791,136 @@ ac_includes_default="\
 # include <unistd.h>
 #endif"
 
-ac_subst_vars='SHELL
-PATH_SEPARATOR
-PACKAGE_NAME
-PACKAGE_TARNAME
-PACKAGE_VERSION
-PACKAGE_STRING
-PACKAGE_BUGREPORT
-exec_prefix
-prefix
-program_transform_name
-bindir
-sbindir
-libexecdir
-datarootdir
-datadir
-sysconfdir
-sharedstatedir
-localstatedir
-includedir
-oldincludedir
-docdir
-infodir
-htmldir
-dvidir
-pdfdir
-psdir
-libdir
-localedir
-mandir
-DEFS
-ECHO_C
-ECHO_N
-ECHO_T
-LIBS
-build_alias
-host_alias
-target_alias
-INSTALL_PROGRAM
-INSTALL_SCRIPT
-INSTALL_DATA
-am__isrc
-CYGPATH_W
-PACKAGE
-VERSION
-ACLOCAL
-AUTOCONF
-AUTOMAKE
-AUTOHEADER
-MAKEINFO
-install_sh
-STRIP
-INSTALL_STRIP_PROGRAM
-MKDIR_P
-mkdir_p
-AWK
-SET_MAKE
-am__leading_dot
-AMTAR
-am__tar
-am__untar
-ARCHIVE_LIBTOOL_VERSION
-BSDCPIO_VERSION_STRING
-BSDTAR_VERSION_STRING
-LIBARCHIVE_VERSION_STRING
-LIBARCHIVE_VERSION_NUMBER
-build
-build_cpu
-build_vendor
-build_os
-host
-host_cpu
-host_vendor
-host_os
-INC_WINDOWS_FILES_TRUE
-INC_WINDOWS_FILES_FALSE
-INC_CYGWIN_FILES_TRUE
-INC_CYGWIN_FILES_FALSE
-CC
-CFLAGS
-LDFLAGS
-CPPFLAGS
-ac_ct_CC
-EXEEXT
-OBJEXT
-DEPDIR
-am__include
-am__quote
-AMDEP_TRUE
-AMDEP_FALSE
-AMDEPBACKSLASH
-CCDEPMODE
-am__fastdepCC_TRUE
-am__fastdepCC_FALSE
-YACC
-YFLAGS
-SED
-GREP
-EGREP
-LN_S
-ECHO
-AR
-RANLIB
-DSYMUTIL
-NMEDIT
-DLLTOOL
-AS
-OBJDUMP
-CPP
-CXX
-CXXFLAGS
-ac_ct_CXX
-CXXDEPMODE
-am__fastdepCXX_TRUE
-am__fastdepCXX_FALSE
-CXXCPP
-F77
-FFLAGS
-ac_ct_F77
-LIBTOOL
-PROG_LDADD_EXTRA
-BUILD_BSDTAR_TRUE
-BUILD_BSDTAR_FALSE
-STATIC_BSDTAR_TRUE
-STATIC_BSDTAR_FALSE
-BUILD_BSDCPIO_TRUE
-BUILD_BSDCPIO_FALSE
-STATIC_BSDCPIO_TRUE
-STATIC_BSDCPIO_FALSE
+ac_subst_vars='LTLIBOBJS
 LIBOBJS
-LTLIBOBJS'
+STATIC_BSDCPIO_FALSE
+STATIC_BSDCPIO_TRUE
+BUILD_BSDCPIO_FALSE
+BUILD_BSDCPIO_TRUE
+STATIC_BSDTAR_FALSE
+STATIC_BSDTAR_TRUE
+BUILD_BSDTAR_FALSE
+BUILD_BSDTAR_TRUE
+PROG_LDADD_EXTRA
+LIBTOOL
+ac_ct_F77
+FFLAGS
+F77
+CXXCPP
+am__fastdepCXX_FALSE
+am__fastdepCXX_TRUE
+CXXDEPMODE
+ac_ct_CXX
+CXXFLAGS
+CXX
+CPP
+OBJDUMP
+AS
+DLLTOOL
+NMEDIT
+DSYMUTIL
+RANLIB
+AR
+ECHO
+LN_S
+EGREP
+GREP
+SED
+YFLAGS
+YACC
+am__fastdepCC_FALSE
+am__fastdepCC_TRUE
+CCDEPMODE
+AMDEPBACKSLASH
+AMDEP_FALSE
+AMDEP_TRUE
+am__quote
+am__include
+DEPDIR
+OBJEXT
+EXEEXT
+ac_ct_CC
+CPPFLAGS
+LDFLAGS
+CFLAGS
+CC
+INC_CYGWIN_FILES_FALSE
+INC_CYGWIN_FILES_TRUE
+INC_WINDOWS_FILES_FALSE
+INC_WINDOWS_FILES_TRUE
+host_os
+host_vendor
+host_cpu
+host
+build_os
+build_vendor
+build_cpu
+build
+LIBARCHIVE_VERSION_NUMBER
+LIBARCHIVE_VERSION_STRING
+BSDTAR_VERSION_STRING
+BSDCPIO_VERSION_STRING
+ARCHIVE_LIBTOOL_VERSION
+am__untar
+am__tar
+AMTAR
+am__leading_dot
+SET_MAKE
+AWK
+mkdir_p
+MKDIR_P
+INSTALL_STRIP_PROGRAM
+STRIP
+install_sh
+MAKEINFO
+AUTOHEADER
+AUTOMAKE
+AUTOCONF
+ACLOCAL
+VERSION
+PACKAGE
+CYGPATH_W
+am__isrc
+INSTALL_DATA
+INSTALL_SCRIPT
+INSTALL_PROGRAM
+target_alias
+host_alias
+build_alias
+LIBS
+ECHO_T
+ECHO_N
+ECHO_C
+DEFS
+mandir
+localedir
+libdir
+psdir
+pdfdir
+dvidir
+htmldir
+infodir
+docdir
+oldincludedir
+includedir
+localstatedir
+sharedstatedir
+sysconfdir
+datadir
+datarootdir
+libexecdir
+sbindir
+bindir
+program_transform_name
+prefix
+exec_prefix
+PACKAGE_BUGREPORT
+PACKAGE_STRING
+PACKAGE_VERSION
+PACKAGE_TARNAME
+PACKAGE_NAME
+PATH_SEPARATOR
+SHELL'
 ac_subst_files=''
 ac_user_opts='
 enable_option_checking
@@ -938,6 +938,7 @@ with_zlib
 with_bz2lib
 with_lzmadec
 with_lzma
+with_openssl
 enable_xattr
 enable_acl
 enable_largefile
@@ -1381,9 +1382,9 @@ fi
 if test -n "$ac_unrecognized_opts"; then
   case $enable_option_checking in
     no) ;;
-    fatal) { $as_echo "$as_me: error: Unrecognized options: $ac_unrecognized_opts" >&2
+    fatal) { $as_echo "$as_me: error: unrecognized options: $ac_unrecognized_opts" >&2
    { (exit 1); exit 1; }; } ;;
-    *)     $as_echo "$as_me: WARNING: Unrecognized options: $ac_unrecognized_opts" >&2 ;;
+    *)     $as_echo "$as_me: WARNING: unrecognized options: $ac_unrecognized_opts" >&2 ;;
   esac
 fi
 
@@ -1436,7 +1437,7 @@ test "$silent" = yes && exec 6>/dev/null
 ac_pwd=`pwd` && test -n "$ac_pwd" &&
 ac_ls_di=`ls -di .` &&
 ac_pwd_ls_di=`cd "$ac_pwd" && ls -di .` ||
-  { $as_echo "$as_me: error: Working directory cannot be determined" >&2
+  { $as_echo "$as_me: error: working directory cannot be determined" >&2
    { (exit 1); exit 1; }; }
 test "X$ac_ls_di" = "X$ac_pwd_ls_di" ||
   { $as_echo "$as_me: error: pwd does not report name of working directory" >&2
@@ -1619,6 +1620,7 @@ Optional Packages:
   --without-bz2lib        Don't build support for bzip2 through bz2lib
   --without-lzmadec       Don't build support for lzma through lzmadec
   --without-lzma          Don't build support for xz through lzma
+  --without-openssl       Don't build support for mtree hashes through openssl
 
 Some influential environment variables:
   CC          C compiler command
@@ -1707,7 +1709,7 @@ test -n "$ac_init_help" && exit $ac_stat
 if $ac_init_version; then
   cat <<\_ACEOF
 libarchive configure 2.7.0
-generated by GNU Autoconf 2.62
+generated by GNU Autoconf 2.63
 
 Copyright (C) 1992, 1993, 1994, 1995, 1996, 1998, 1999, 2000, 2001,
 2002, 2003, 2004, 2005, 2006, 2007, 2008 Free Software Foundation, Inc.
@@ -1721,7 +1723,7 @@ This file contains any messages produced
 running configure, to aid debugging if configure makes a mistake.
 
 It was created by libarchive $as_me 2.7.0, which was
-generated by GNU Autoconf 2.62.  Invocation command line was
+generated by GNU Autoconf 2.63.  Invocation command line was
 
   $ $0 $@
 
@@ -1844,8 +1846,8 @@ _ASBOX
     case $ac_val in #(
     *${as_nl}*)
       case $ac_var in #(
-      *_cv_*) { $as_echo "$as_me:$LINENO: WARNING: Cache variable $ac_var contains a newline." >&5
-$as_echo "$as_me: WARNING: Cache variable $ac_var contains a newline." >&2;} ;;
+      *_cv_*) { $as_echo "$as_me:$LINENO: WARNING: cache variable $ac_var contains a newline" >&5
+$as_echo "$as_me: WARNING: cache variable $ac_var contains a newline" >&2;} ;;
       esac
       case $ac_var in #(
       _ | IFS | as_nl) ;; #(
@@ -2048,6 +2050,8 @@ $as_echo "$as_me:   current value: \`$ac
   fi
 done
 if $ac_cache_corrupted; then
+  { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
   { $as_echo "$as_me:$LINENO: error: changes in the environment can compromise the build" >&5
 $as_echo "$as_me: error: changes in the environment can compromise the build" >&2;}
   { { $as_echo "$as_me:$LINENO: error: run \`make distclean' and/or \`rm $cache_file' and start over" >&5
@@ -2557,12 +2561,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     STRIP=$ac_ct_STRIP
@@ -2837,12 +2837,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     CC=$ac_ct_CC
@@ -3041,12 +3037,8 @@ done
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     CC=$ac_ct_CC
@@ -3056,11 +3048,13 @@ fi
 fi
 
 
-test -z "$CC" && { { $as_echo "$as_me:$LINENO: error: no acceptable C compiler found in \$PATH
+test -z "$CC" && { { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: no acceptable C compiler found in \$PATH
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: no acceptable C compiler found in \$PATH
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 
 # Provide some information about the compiler.
 $as_echo "$as_me:$LINENO: checking for C compiler version" >&5
@@ -3190,11 +3184,13 @@ if test -z "$ac_file"; then
   $as_echo "$as_me: failed program was:" >&5
 sed 's/^/| /' conftest.$ac_ext >&5
 
+{ { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
 { { $as_echo "$as_me:$LINENO: error: C compiler cannot create executables
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: C compiler cannot create executables
 See \`config.log' for more details." >&2;}
-   { (exit 77); exit 77; }; }
+   { (exit 77); exit 77; }; }; }
 fi
 
 ac_exeext=$ac_cv_exeext
@@ -3222,13 +3218,15 @@ $as_echo "$ac_try_echo") >&5
     if test "$cross_compiling" = maybe; then
 	cross_compiling=yes
     else
-	{ { $as_echo "$as_me:$LINENO: error: cannot run C compiled programs.
+	{ { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: cannot run C compiled programs.
 If you meant to cross compile, use \`--host'.
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: cannot run C compiled programs.
 If you meant to cross compile, use \`--host'.
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
     fi
   fi
 fi
@@ -3271,11 +3269,13 @@ for ac_file in conftest.exe conftest con
   esac
 done
 else
-  { { $as_echo "$as_me:$LINENO: error: cannot compute suffix of executables: cannot compile and link
+  { { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: cannot compute suffix of executables: cannot compile and link
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: cannot compute suffix of executables: cannot compile and link
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 fi
 
 rm -f conftest$ac_cv_exeext
@@ -3329,11 +3329,13 @@ else
   $as_echo "$as_me: failed program was:" >&5
 sed 's/^/| /' conftest.$ac_ext >&5
 
+{ { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
 { { $as_echo "$as_me:$LINENO: error: cannot compute suffix of object files: cannot compile
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: cannot compute suffix of object files: cannot compile
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 fi
 
 rm -f conftest.$ac_cv_objext conftest.$ac_ext
@@ -4724,7 +4726,7 @@ ia64-*-hpux*)
   ;;
 *-*-irix6*)
   # Find out which ABI we are using.
-  echo '#line 4727 "configure"' > conftest.$ac_ext
+  echo '#line 4729 "configure"' > conftest.$ac_ext
   if { (eval echo "$as_me:$LINENO: \"$ac_compile\"") >&5
   (eval $ac_compile) 2>&5
   ac_status=$?
@@ -4789,6 +4791,7 @@ s390*-*linux*|sparc*-*linux*)
       esac
       ;;
     *64-bit*)
+      libsuff=64
       case $host in
         x86_64-*kfreebsd*-gnu)
           LD="${LD-ld} -m elf_x86_64_fbsd"
@@ -4995,12 +4998,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     DLLTOOL=$ac_ct_DLLTOOL
@@ -5091,12 +5090,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     AS=$ac_ct_AS
@@ -5187,12 +5182,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     OBJDUMP=$ac_ct_OBJDUMP
@@ -5429,11 +5420,13 @@ rm -f conftest.err conftest.$ac_ext
 if $ac_preproc_ok; then
   :
 else
-  { { $as_echo "$as_me:$LINENO: error: C preprocessor \"$CPP\" fails sanity check
+  { { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: C preprocessor \"$CPP\" fails sanity check
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: C preprocessor \"$CPP\" fails sanity check
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 fi
 
 ac_ext=c
@@ -5682,8 +5675,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -5833,8 +5827,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -5944,12 +5939,8 @@ done
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     CXX=$ac_ct_CXX
@@ -6549,11 +6540,13 @@ rm -f conftest.err conftest.$ac_ext
 if $ac_preproc_ok; then
   :
 else
-  { { $as_echo "$as_me:$LINENO: error: C++ preprocessor \"$CXXCPP\" fails sanity check
+  { { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: C++ preprocessor \"$CXXCPP\" fails sanity check
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: C++ preprocessor \"$CXXCPP\" fails sanity check
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 fi
 
 ac_ext=cpp
@@ -6661,12 +6654,8 @@ done
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     F77=$ac_ct_F77
@@ -7321,12 +7310,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     AR=$ac_ct_AR
@@ -7417,12 +7402,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     RANLIB=$ac_ct_RANLIB
@@ -7513,12 +7494,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     STRIP=$ac_ct_STRIP
@@ -7797,12 +7774,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     DSYMUTIL=$ac_ct_DSYMUTIL
@@ -7893,12 +7866,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     NMEDIT=$ac_ct_NMEDIT
@@ -8121,11 +8090,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:8124: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:8093: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>conftest.err)
    ac_status=$?
    cat conftest.err >&5
-   echo "$as_me:8128: \$? = $ac_status" >&5
+   echo "$as_me:8097: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s "$ac_outfile"; then
      # The compiler can only warn and ignore the option if not recognized
      # So say no if there are warnings other than the usual output.
@@ -8411,11 +8380,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:8414: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:8383: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>conftest.err)
    ac_status=$?
    cat conftest.err >&5
-   echo "$as_me:8418: \$? = $ac_status" >&5
+   echo "$as_me:8387: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s "$ac_outfile"; then
      # The compiler can only warn and ignore the option if not recognized
      # So say no if there are warnings other than the usual output.
@@ -8515,11 +8484,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:8518: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:8487: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>out/conftest.err)
    ac_status=$?
    cat out/conftest.err >&5
-   echo "$as_me:8522: \$? = $ac_status" >&5
+   echo "$as_me:8491: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s out/conftest2.$ac_objext
    then
      # The compiler can only warn and ignore the option if not recognized
@@ -10031,11 +10000,13 @@ linux* | k*bsd*-gnu)
   # Some rework will be needed to allow for fast_install
   # before this can be enabled.
   hardcode_into_libs=yes
+  sys_lib_search_path_spec="/usr/lib${libsuff} /lib${libsuff} /usr/local/lib${libsuff}"
+  sys_lib_dlsearch_path_spec="/usr/lib${libsuff} /lib${libsuff}"
 
   # Append ld.so.conf contents to the search path
   if test -f /etc/ld.so.conf; then
     lt_ld_extra=`awk '/^include / { system(sprintf("cd /etc; cat %s 2>/dev/null", \$2)); skip = 1; } { if (!skip) print \$0; skip = 0; }' < /etc/ld.so.conf | $SED -e 's/#.*//;/^[ 	]*hwcap[ 	]/d;s/[:,	]/ /g;s/=[^=]*$//;s/=[^= ]* / /g;/^$/d' | tr '\n' ' '`
-    sys_lib_dlsearch_path_spec="/lib /usr/lib $lt_ld_extra"
+    sys_lib_dlsearch_path_spec="$sys_lib_dlsearch_path_spec $lt_ld_extra"
   fi
 
   # We used to test for /lib/ld.so.1 and disable shared libraries on
@@ -10407,7 +10378,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_dl_dlopen" >&5
 $as_echo "$ac_cv_lib_dl_dlopen" >&6; }
-if test $ac_cv_lib_dl_dlopen = yes; then
+if test "x$ac_cv_lib_dl_dlopen" = x""yes; then
   lt_cv_dlopen="dlopen" lt_cv_dlopen_libs="-ldl"
 else
 
@@ -10505,7 +10476,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_shl_load" >&5
 $as_echo "$ac_cv_func_shl_load" >&6; }
-if test $ac_cv_func_shl_load = yes; then
+if test "x$ac_cv_func_shl_load" = x""yes; then
   lt_cv_dlopen="shl_load"
 else
   { $as_echo "$as_me:$LINENO: checking for shl_load in -ldld" >&5
@@ -10573,7 +10544,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_dld_shl_load" >&5
 $as_echo "$ac_cv_lib_dld_shl_load" >&6; }
-if test $ac_cv_lib_dld_shl_load = yes; then
+if test "x$ac_cv_lib_dld_shl_load" = x""yes; then
   lt_cv_dlopen="shl_load" lt_cv_dlopen_libs="-ldld"
 else
   { $as_echo "$as_me:$LINENO: checking for dlopen" >&5
@@ -10661,7 +10632,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_dlopen" >&5
 $as_echo "$ac_cv_func_dlopen" >&6; }
-if test $ac_cv_func_dlopen = yes; then
+if test "x$ac_cv_func_dlopen" = x""yes; then
   lt_cv_dlopen="dlopen"
 else
   { $as_echo "$as_me:$LINENO: checking for dlopen in -ldl" >&5
@@ -10729,7 +10700,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_dl_dlopen" >&5
 $as_echo "$ac_cv_lib_dl_dlopen" >&6; }
-if test $ac_cv_lib_dl_dlopen = yes; then
+if test "x$ac_cv_lib_dl_dlopen" = x""yes; then
   lt_cv_dlopen="dlopen" lt_cv_dlopen_libs="-ldl"
 else
   { $as_echo "$as_me:$LINENO: checking for dlopen in -lsvld" >&5
@@ -10797,7 +10768,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_svld_dlopen" >&5
 $as_echo "$ac_cv_lib_svld_dlopen" >&6; }
-if test $ac_cv_lib_svld_dlopen = yes; then
+if test "x$ac_cv_lib_svld_dlopen" = x""yes; then
   lt_cv_dlopen="dlopen" lt_cv_dlopen_libs="-lsvld"
 else
   { $as_echo "$as_me:$LINENO: checking for dld_link in -ldld" >&5
@@ -10865,7 +10836,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_dld_dld_link" >&5
 $as_echo "$ac_cv_lib_dld_dld_link" >&6; }
-if test $ac_cv_lib_dld_dld_link = yes; then
+if test "x$ac_cv_lib_dld_dld_link" = x""yes; then
   lt_cv_dlopen="dld_link" lt_cv_dlopen_libs="-ldld"
 fi
 
@@ -10915,7 +10886,7 @@ else
   lt_dlunknown=0; lt_dlno_uscore=1; lt_dlneed_uscore=2
   lt_status=$lt_dlunknown
   cat > conftest.$ac_ext <<EOF
-#line 10918 "configure"
+#line 10889 "configure"
 #include "confdefs.h"
 
 #if HAVE_DLFCN_H
@@ -11015,7 +10986,7 @@ else
   lt_dlunknown=0; lt_dlno_uscore=1; lt_dlneed_uscore=2
   lt_status=$lt_dlunknown
   cat > conftest.$ac_ext <<EOF
-#line 11018 "configure"
+#line 10989 "configure"
 #include "confdefs.h"
 
 #if HAVE_DLFCN_H
@@ -13424,11 +13395,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:13427: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:13398: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>conftest.err)
    ac_status=$?
    cat conftest.err >&5
-   echo "$as_me:13431: \$? = $ac_status" >&5
+   echo "$as_me:13402: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s "$ac_outfile"; then
      # The compiler can only warn and ignore the option if not recognized
      # So say no if there are warnings other than the usual output.
@@ -13528,11 +13499,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:13531: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:13502: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>out/conftest.err)
    ac_status=$?
    cat out/conftest.err >&5
-   echo "$as_me:13535: \$? = $ac_status" >&5
+   echo "$as_me:13506: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s out/conftest2.$ac_objext
    then
      # The compiler can only warn and ignore the option if not recognized
@@ -14030,11 +14001,13 @@ linux* | k*bsd*-gnu)
   # Some rework will be needed to allow for fast_install
   # before this can be enabled.
   hardcode_into_libs=yes
+  sys_lib_search_path_spec="/usr/lib${libsuff} /lib${libsuff} /usr/local/lib${libsuff}"
+  sys_lib_dlsearch_path_spec="/usr/lib${libsuff} /lib${libsuff}"
 
   # Append ld.so.conf contents to the search path
   if test -f /etc/ld.so.conf; then
     lt_ld_extra=`awk '/^include / { system(sprintf("cd /etc; cat %s 2>/dev/null", \$2)); skip = 1; } { if (!skip) print \$0; skip = 0; }' < /etc/ld.so.conf | $SED -e 's/#.*//;/^[ 	]*hwcap[ 	]/d;s/[:,	]/ /g;s/=[^=]*$//;s/=[^= ]* / /g;/^$/d' | tr '\n' ' '`
-    sys_lib_dlsearch_path_spec="/lib /usr/lib $lt_ld_extra"
+    sys_lib_dlsearch_path_spec="$sys_lib_dlsearch_path_spec $lt_ld_extra"
   fi
 
   # We used to test for /lib/ld.so.1 and disable shared libraries on
@@ -15111,11 +15084,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:15114: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:15087: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>conftest.err)
    ac_status=$?
    cat conftest.err >&5
-   echo "$as_me:15118: \$? = $ac_status" >&5
+   echo "$as_me:15091: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s "$ac_outfile"; then
      # The compiler can only warn and ignore the option if not recognized
      # So say no if there are warnings other than the usual output.
@@ -15215,11 +15188,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:15218: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:15191: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>out/conftest.err)
    ac_status=$?
    cat out/conftest.err >&5
-   echo "$as_me:15222: \$? = $ac_status" >&5
+   echo "$as_me:15195: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s out/conftest2.$ac_objext
    then
      # The compiler can only warn and ignore the option if not recognized
@@ -16659,11 +16632,13 @@ linux* | k*bsd*-gnu)
   # Some rework will be needed to allow for fast_install
   # before this can be enabled.
   hardcode_into_libs=yes
+  sys_lib_search_path_spec="/usr/lib${libsuff} /lib${libsuff} /usr/local/lib${libsuff}"
+  sys_lib_dlsearch_path_spec="/usr/lib${libsuff} /lib${libsuff}"
 
   # Append ld.so.conf contents to the search path
   if test -f /etc/ld.so.conf; then
     lt_ld_extra=`awk '/^include / { system(sprintf("cd /etc; cat %s 2>/dev/null", \$2)); skip = 1; } { if (!skip) print \$0; skip = 0; }' < /etc/ld.so.conf | $SED -e 's/#.*//;/^[ 	]*hwcap[ 	]/d;s/[:,	]/ /g;s/=[^=]*$//;s/=[^= ]* / /g;/^$/d' | tr '\n' ' '`
-    sys_lib_dlsearch_path_spec="/lib /usr/lib $lt_ld_extra"
+    sys_lib_dlsearch_path_spec="$sys_lib_dlsearch_path_spec $lt_ld_extra"
   fi
 
   # We used to test for /lib/ld.so.1 and disable shared libraries on
@@ -17430,11 +17405,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:17433: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:17408: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>conftest.err)
    ac_status=$?
    cat conftest.err >&5
-   echo "$as_me:17437: \$? = $ac_status" >&5
+   echo "$as_me:17412: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s "$ac_outfile"; then
      # The compiler can only warn and ignore the option if not recognized
      # So say no if there are warnings other than the usual output.
@@ -17720,11 +17695,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:17723: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:17698: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>conftest.err)
    ac_status=$?
    cat conftest.err >&5
-   echo "$as_me:17727: \$? = $ac_status" >&5
+   echo "$as_me:17702: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s "$ac_outfile"; then
      # The compiler can only warn and ignore the option if not recognized
      # So say no if there are warnings other than the usual output.
@@ -17824,11 +17799,11 @@ else
    -e 's:.*FLAGS}\{0,1\} :&$lt_compiler_flag :; t' \
    -e 's: [^ ]*conftest\.: $lt_compiler_flag&:; t' \
    -e 's:$: $lt_compiler_flag:'`
-   (eval echo "\"\$as_me:17827: $lt_compile\"" >&5)
+   (eval echo "\"\$as_me:17802: $lt_compile\"" >&5)
    (eval "$lt_compile" 2>out/conftest.err)
    ac_status=$?
    cat out/conftest.err >&5
-   echo "$as_me:17831: \$? = $ac_status" >&5
+   echo "$as_me:17806: \$? = $ac_status" >&5
    if (exit $ac_status) && test -s out/conftest2.$ac_objext
    then
      # The compiler can only warn and ignore the option if not recognized
@@ -19288,11 +19263,13 @@ linux* | k*bsd*-gnu)
   # Some rework will be needed to allow for fast_install
   # before this can be enabled.
   hardcode_into_libs=yes
+  sys_lib_search_path_spec="/usr/lib${libsuff} /lib${libsuff} /usr/local/lib${libsuff}"
+  sys_lib_dlsearch_path_spec="/usr/lib${libsuff} /lib${libsuff}"
 
   # Append ld.so.conf contents to the search path
   if test -f /etc/ld.so.conf; then
     lt_ld_extra=`awk '/^include / { system(sprintf("cd /etc; cat %s 2>/dev/null", \$2)); skip = 1; } { if (!skip) print \$0; skip = 0; }' < /etc/ld.so.conf | $SED -e 's/#.*//;/^[ 	]*hwcap[ 	]/d;s/[:,	]/ /g;s/=[^=]*$//;s/=[^= ]* / /g;/^$/d' | tr '\n' ' '`
-    sys_lib_dlsearch_path_spec="/lib /usr/lib $lt_ld_extra"
+    sys_lib_dlsearch_path_spec="$sys_lib_dlsearch_path_spec $lt_ld_extra"
   fi
 
   # We used to test for /lib/ld.so.1 and disable shared libraries on
@@ -20591,12 +20568,8 @@ fi
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
-{ $as_echo "$as_me:$LINENO: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&5
-$as_echo "$as_me: WARNING: In the future, Autoconf will not detect cross-tools
-whose name does not start with the host triplet.  If you think this
-configuration is useful to you, please write to autoconf@gnu.org." >&2;}
+{ $as_echo "$as_me:$LINENO: WARNING: using cross tools not prefixed with host triplet" >&5
+$as_echo "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
 ac_tool_warned=yes ;;
 esac
     STRIP=$ac_ct_STRIP
@@ -20645,11 +20618,13 @@ yes)
 	;;
 dynamic|shared)
 	if test "$enable_shared" = "no"; then
-		{ { $as_echo "$as_me:$LINENO: error: Shared linking of bsdtar requires shared libarchive
+		{ { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: Shared linking of bsdtar requires shared libarchive
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: Shared linking of bsdtar requires shared libarchive
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 	fi
 	build_bsdtar=yes
 	static_bsdtar=no
@@ -20663,11 +20638,13 @@ no)
 	static_bsdtar=no
 	;;
 *)
-	{ { $as_echo "$as_me:$LINENO: error: Unsupported value for --enable-bsdtar
+	{ { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: Unsupported value for --enable-bsdtar
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: Unsupported value for --enable-bsdtar
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 	;;
 esac
 
@@ -20712,11 +20689,13 @@ yes)
 	;;
 dynamic|shared)
 	if test "$enabled_shared" = "no"; then
-	   { { $as_echo "$as_me:$LINENO: error: Shared linking of bsdcpio requires shared libarchive
+	   { { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: Shared linking of bsdcpio requires shared libarchive
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: Shared linking of bsdcpio requires shared libarchive
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 	fi
 	build_bsdcpio=yes
 	;;
@@ -20729,11 +20708,13 @@ no)
 	static_bsdcpio=no
 	;;
 *)
-	{ { $as_echo "$as_me:$LINENO: error: Unsupported value for --enable-bsdcpio
+	{ { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: Unsupported value for --enable-bsdcpio
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: Unsupported value for --enable-bsdcpio
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); exit 1; }; }; }
 	;;
 esac
 
@@ -20996,8 +20977,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_hdr" | $as_tr_cpp` 1
 _ACEOF
@@ -21396,8 +21378,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -21548,8 +21531,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -21701,8 +21685,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -21857,8 +21842,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -22009,8 +21995,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -22162,8 +22149,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -22316,8 +22304,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -22471,8 +22460,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -22631,8 +22621,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -22707,7 +22698,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_z_inflate" >&5
 $as_echo "$ac_cv_lib_z_inflate" >&6; }
-if test $ac_cv_lib_z_inflate = yes; then
+if test "x$ac_cv_lib_z_inflate" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBZ 1
 _ACEOF
@@ -22866,8 +22857,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -22942,7 +22934,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_bz2_BZ2_bzDecompressInit" >&5
 $as_echo "$ac_cv_lib_bz2_BZ2_bzDecompressInit" >&6; }
-if test $ac_cv_lib_bz2_BZ2_bzDecompressInit = yes; then
+if test "x$ac_cv_lib_bz2_BZ2_bzDecompressInit" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBBZ2 1
 _ACEOF
@@ -23101,8 +23093,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -23177,7 +23170,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_lzmadec_lzmadec_decode" >&5
 $as_echo "$ac_cv_lib_lzmadec_lzmadec_decode" >&6; }
-if test $ac_cv_lib_lzmadec_lzmadec_decode = yes; then
+if test "x$ac_cv_lib_lzmadec_lzmadec_decode" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBLZMADEC 1
 _ACEOF
@@ -23336,8 +23329,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -23412,7 +23406,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_lzma_lzma_stream_decoder" >&5
 $as_echo "$ac_cv_lib_lzma_lzma_stream_decoder" >&6; }
-if test $ac_cv_lib_lzma_lzma_stream_decoder = yes; then
+if test "x$ac_cv_lib_lzma_lzma_stream_decoder" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBLZMA 1
 _ACEOF
@@ -23424,6 +23418,14 @@ fi
 fi
 
 
+# Check whether --with-openssl was given.
+if test "${with_openssl+set}" = set; then
+  withval=$with_openssl;
+fi
+
+
+if test "x$with_openssl" != "xno"; then
+
 
 
 for ac_header in openssl/md5.h openssl/ripemd.h openssl/sha.h
@@ -23565,8 +23567,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -23715,8 +23718,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -23866,8 +23870,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -24019,8 +24024,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -24029,7 +24035,7 @@ fi
 
 done
 
-{ $as_echo "$as_me:$LINENO: checking for library containing SHA1_Init" >&5
+  { $as_echo "$as_me:$LINENO: checking for library containing SHA1_Init" >&5
 $as_echo_n "checking for library containing SHA1_Init... " >&6; }
 if test "${ac_cv_search_SHA1_Init+set}" = set; then
   $as_echo_n "(cached) " >&6
@@ -24116,7 +24122,7 @@ if test "$ac_res" != no; then
 
 fi
 
-{ $as_echo "$as_me:$LINENO: checking for MD5_Init" >&5
+  { $as_echo "$as_me:$LINENO: checking for MD5_Init" >&5
 $as_echo_n "checking for MD5_Init... " >&6; }
 if test "${ac_cv_func_MD5_Init+set}" = set; then
   $as_echo_n "(cached) " >&6
@@ -24202,7 +24208,7 @@ fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_MD5_Init" >&5
 $as_echo "$ac_cv_func_MD5_Init" >&6; }
 
-if test "x$ac_cv_func_MD5_Init" != "xyes"; then
+  if test "x$ac_cv_func_MD5_Init" != "xyes"; then
   { $as_echo "$as_me:$LINENO: checking for MD5Init" >&5
 $as_echo_n "checking for MD5Init... " >&6; }
 if test "${ac_cv_func_MD5Init+set}" = set; then
@@ -24288,7 +24294,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_MD5Init" >&5
 $as_echo "$ac_cv_func_MD5Init" >&6; }
-if test $ac_cv_func_MD5Init = yes; then
+if test "x$ac_cv_func_MD5Init" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define MD5_Init MD5Init
@@ -24307,15 +24313,15 @@ _ACEOF
 
 fi
 
-fi
-if test "x$ac_cv_func_MD5_Init" = "xyes" -o "x$ac_cv_func_MD5Init" = "xyes" ; then
+  fi
+  if test "x$ac_cv_func_MD5_Init" = "xyes" -o "x$ac_cv_func_MD5Init" = "xyes" ; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_MD5 1
 _ACEOF
 
-fi
-{ $as_echo "$as_me:$LINENO: checking for RIPEMD160_Init" >&5
+  fi
+  { $as_echo "$as_me:$LINENO: checking for RIPEMD160_Init" >&5
 $as_echo_n "checking for RIPEMD160_Init... " >&6; }
 if test "${ac_cv_func_RIPEMD160_Init+set}" = set; then
   $as_echo_n "(cached) " >&6
@@ -24401,7 +24407,7 @@ fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_RIPEMD160_Init" >&5
 $as_echo "$ac_cv_func_RIPEMD160_Init" >&6; }
 
-if test "x$ac_cv_func_RIPEMD160_Init" != "xyes"; then
+  if test "x$ac_cv_func_RIPEMD160_Init" != "xyes"; then
   { $as_echo "$as_me:$LINENO: checking for RMD160Init" >&5
 $as_echo_n "checking for RMD160Init... " >&6; }
 if test "${ac_cv_func_RMD160Init+set}" = set; then
@@ -24487,7 +24493,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_RMD160Init" >&5
 $as_echo "$ac_cv_func_RMD160Init" >&6; }
-if test $ac_cv_func_RMD160Init = yes; then
+if test "x$ac_cv_func_RMD160Init" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define RIPEMD160_Init RMD160Init
@@ -24506,15 +24512,15 @@ _ACEOF
 
 fi
 
-fi
-if test "x$ac_cv_func_RIPEMD160_Init" = "xyes" -o "x$ac_cv_func_RMD160Init" = "xyes" ; then
+  fi
+  if test "x$ac_cv_func_RIPEMD160_Init" = "xyes" -o "x$ac_cv_func_RMD160Init" = "xyes" ; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_RMD160 1
 _ACEOF
 
-fi
-{ $as_echo "$as_me:$LINENO: checking for SHA1_Init" >&5
+  fi
+  { $as_echo "$as_me:$LINENO: checking for SHA1_Init" >&5
 $as_echo_n "checking for SHA1_Init... " >&6; }
 if test "${ac_cv_func_SHA1_Init+set}" = set; then
   $as_echo_n "(cached) " >&6
@@ -24600,7 +24606,7 @@ fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_SHA1_Init" >&5
 $as_echo "$ac_cv_func_SHA1_Init" >&6; }
 
-if test "x$ac_cv_func_SHA1_Init" != "xyes"; then
+  if test "x$ac_cv_func_SHA1_Init" != "xyes"; then
   { $as_echo "$as_me:$LINENO: checking for SHA1Init" >&5
 $as_echo_n "checking for SHA1Init... " >&6; }
 if test "${ac_cv_func_SHA1Init+set}" = set; then
@@ -24686,7 +24692,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_SHA1Init" >&5
 $as_echo "$ac_cv_func_SHA1Init" >&6; }
-if test $ac_cv_func_SHA1Init = yes; then
+if test "x$ac_cv_func_SHA1Init" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define SHA1_Init SHA1Init
@@ -24705,15 +24711,15 @@ _ACEOF
 
 fi
 
-fi
-if test "x$ac_cv_func_SHA1_Init" = "xyes" -o "x$ac_cv_func_SHA1Init" = "xyes" ; then
+  fi
+  if test "x$ac_cv_func_SHA1_Init" = "xyes" -o "x$ac_cv_func_SHA1Init" = "xyes" ; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_SHA1 1
 _ACEOF
 
-fi
-{ $as_echo "$as_me:$LINENO: checking for SHA256_Init" >&5
+  fi
+  { $as_echo "$as_me:$LINENO: checking for SHA256_Init" >&5
 $as_echo_n "checking for SHA256_Init... " >&6; }
 if test "${ac_cv_func_SHA256_Init+set}" = set; then
   $as_echo_n "(cached) " >&6
@@ -24799,14 +24805,14 @@ fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_SHA256_Init" >&5
 $as_echo "$ac_cv_func_SHA256_Init" >&6; }
 
-if test "x$ac_cv_func_SHA256_Init" = "xyes" ; then
+  if test "x$ac_cv_func_SHA256_Init" = "xyes" ; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_SHA256 1
 _ACEOF
 
-fi
-{ $as_echo "$as_me:$LINENO: checking for SHA384_Init" >&5
+  fi
+  { $as_echo "$as_me:$LINENO: checking for SHA384_Init" >&5
 $as_echo_n "checking for SHA384_Init... " >&6; }
 if test "${ac_cv_func_SHA384_Init+set}" = set; then
   $as_echo_n "(cached) " >&6
@@ -24892,14 +24898,14 @@ fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_SHA384_Init" >&5
 $as_echo "$ac_cv_func_SHA384_Init" >&6; }
 
-if test "x$ac_cv_func_SHA384_Init" = "xyes" ; then
+  if test "x$ac_cv_func_SHA384_Init" = "xyes" ; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_SHA384 1
 _ACEOF
 
-fi
-{ $as_echo "$as_me:$LINENO: checking for SHA512_Init" >&5
+  fi
+  { $as_echo "$as_me:$LINENO: checking for SHA512_Init" >&5
 $as_echo_n "checking for SHA512_Init... " >&6; }
 if test "${ac_cv_func_SHA512_Init+set}" = set; then
   $as_echo_n "(cached) " >&6
@@ -24985,13 +24991,14 @@ fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func_SHA512_Init" >&5
 $as_echo "$ac_cv_func_SHA512_Init" >&6; }
 
-if test "x$ac_cv_func_SHA512_Init" = "xyes" ; then
+  if test "x$ac_cv_func_SHA512_Init" = "xyes" ; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_SHA512 1
 _ACEOF
 
-fi
+  fi
+fi # with_openssl
 
 # TODO: Give the user the option of using a pre-existing system
 # libarchive.  This will define HAVE_LIBARCHIVE which will cause
@@ -25240,7 +25247,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_mode_t" >&5
 $as_echo "$ac_cv_type_mode_t" >&6; }
-if test $ac_cv_type_mode_t = yes; then
+if test "x$ac_cv_type_mode_t" = x""yes; then
   :
 else
 
@@ -25346,7 +25353,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_off_t" >&5
 $as_echo "$ac_cv_type_off_t" >&6; }
-if test $ac_cv_type_off_t = yes; then
+if test "x$ac_cv_type_off_t" = x""yes; then
   :
 else
 
@@ -25450,7 +25457,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_size_t" >&5
 $as_echo "$ac_cv_type_size_t" >&6; }
-if test $ac_cv_type_size_t = yes; then
+if test "x$ac_cv_type_size_t" = x""yes; then
   :
 else
 
@@ -25554,7 +25561,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_id_t" >&5
 $as_echo "$ac_cv_type_id_t" >&6; }
-if test $ac_cv_type_id_t = yes; then
+if test "x$ac_cv_type_id_t" = x""yes; then
   :
 else
 
@@ -25658,7 +25665,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_uintptr_t" >&5
 $as_echo "$ac_cv_type_uintptr_t" >&6; }
-if test $ac_cv_type_uintptr_t = yes; then
+if test "x$ac_cv_type_uintptr_t" = x""yes; then
   :
 else
 
@@ -25765,7 +25772,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_birthtime" >&5
 $as_echo "$ac_cv_member_struct_stat_st_birthtime" >&6; }
-if test $ac_cv_member_struct_stat_st_birthtime = yes; then
+if test "x$ac_cv_member_struct_stat_st_birthtime" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_BIRTHTIME 1
@@ -25871,7 +25878,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_birthtimespec_tv_nsec" >&5
 $as_echo "$ac_cv_member_struct_stat_st_birthtimespec_tv_nsec" >&6; }
-if test $ac_cv_member_struct_stat_st_birthtimespec_tv_nsec = yes; then
+if test "x$ac_cv_member_struct_stat_st_birthtimespec_tv_nsec" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_BIRTHTIMESPEC_TV_NSEC 1
@@ -25975,7 +25982,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_mtimespec_tv_nsec" >&5
 $as_echo "$ac_cv_member_struct_stat_st_mtimespec_tv_nsec" >&6; }
-if test $ac_cv_member_struct_stat_st_mtimespec_tv_nsec = yes; then
+if test "x$ac_cv_member_struct_stat_st_mtimespec_tv_nsec" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_MTIMESPEC_TV_NSEC 1
@@ -26079,7 +26086,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_mtim_tv_nsec" >&5
 $as_echo "$ac_cv_member_struct_stat_st_mtim_tv_nsec" >&6; }
-if test $ac_cv_member_struct_stat_st_mtim_tv_nsec = yes; then
+if test "x$ac_cv_member_struct_stat_st_mtim_tv_nsec" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_MTIM_TV_NSEC 1
@@ -26183,7 +26190,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_mtime_n" >&5
 $as_echo "$ac_cv_member_struct_stat_st_mtime_n" >&6; }
-if test $ac_cv_member_struct_stat_st_mtime_n = yes; then
+if test "x$ac_cv_member_struct_stat_st_mtime_n" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_MTIME_N 1
@@ -26287,7 +26294,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_umtime" >&5
 $as_echo "$ac_cv_member_struct_stat_st_umtime" >&6; }
-if test $ac_cv_member_struct_stat_st_umtime = yes; then
+if test "x$ac_cv_member_struct_stat_st_umtime" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_UMTIME 1
@@ -26391,7 +26398,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_mtime_usec" >&5
 $as_echo "$ac_cv_member_struct_stat_st_mtime_usec" >&6; }
-if test $ac_cv_member_struct_stat_st_mtime_usec = yes; then
+if test "x$ac_cv_member_struct_stat_st_mtime_usec" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_MTIME_USEC 1
@@ -26496,7 +26503,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_blksize" >&5
 $as_echo "$ac_cv_member_struct_stat_st_blksize" >&6; }
-if test $ac_cv_member_struct_stat_st_blksize = yes; then
+if test "x$ac_cv_member_struct_stat_st_blksize" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_BLKSIZE 1
@@ -26601,7 +26608,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_member_struct_stat_st_flags" >&5
 $as_echo "$ac_cv_member_struct_stat_st_flags" >&6; }
-if test $ac_cv_member_struct_stat_st_flags = yes; then
+if test "x$ac_cv_member_struct_stat_st_flags" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_STRUCT_STAT_ST_FLAGS 1
@@ -26708,7 +26715,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_uintmax_t" >&5
 $as_echo "$ac_cv_type_uintmax_t" >&6; }
-if test $ac_cv_type_uintmax_t = yes; then
+if test "x$ac_cv_type_uintmax_t" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_UINTMAX_T 1
@@ -26810,7 +26817,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_unsigned_long_long" >&5
 $as_echo "$ac_cv_type_unsigned_long_long" >&6; }
-if test $ac_cv_type_unsigned_long_long = yes; then
+if test "x$ac_cv_type_unsigned_long_long" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_UNSIGNED_LONG_LONG 1
@@ -27070,7 +27077,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_intmax_t" >&5
 $as_echo "$ac_cv_type_intmax_t" >&6; }
-if test $ac_cv_type_intmax_t = yes; then
+if test "x$ac_cv_type_intmax_t" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_INTMAX_T 1
@@ -27385,7 +27392,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_uintmax_t" >&5
 $as_echo "$ac_cv_type_uintmax_t" >&6; }
-if test $ac_cv_type_uintmax_t = yes; then
+if test "x$ac_cv_type_uintmax_t" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_UINTMAX_T 1
@@ -27535,7 +27542,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_SIZE_MAX" >&5
 $as_echo "$ac_cv_have_decl_SIZE_MAX" >&6; }
-if test $ac_cv_have_decl_SIZE_MAX = yes; then
+if test "x$ac_cv_have_decl_SIZE_MAX" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_SIZE_MAX 1
@@ -27602,7 +27609,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_SSIZE_MAX" >&5
 $as_echo "$ac_cv_have_decl_SSIZE_MAX" >&6; }
-if test $ac_cv_have_decl_SSIZE_MAX = yes; then
+if test "x$ac_cv_have_decl_SSIZE_MAX" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_SSIZE_MAX 1
@@ -27669,7 +27676,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_INT64_MAX" >&5
 $as_echo "$ac_cv_have_decl_INT64_MAX" >&6; }
-if test $ac_cv_have_decl_INT64_MAX = yes; then
+if test "x$ac_cv_have_decl_INT64_MAX" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_INT64_MAX 1
@@ -27736,7 +27743,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_INT64_MIN" >&5
 $as_echo "$ac_cv_have_decl_INT64_MIN" >&6; }
-if test $ac_cv_have_decl_INT64_MIN = yes; then
+if test "x$ac_cv_have_decl_INT64_MIN" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_INT64_MIN 1
@@ -27803,7 +27810,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_UINT64_MAX" >&5
 $as_echo "$ac_cv_have_decl_UINT64_MAX" >&6; }
-if test $ac_cv_have_decl_UINT64_MAX = yes; then
+if test "x$ac_cv_have_decl_UINT64_MAX" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_UINT64_MAX 1
@@ -27870,7 +27877,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_UINT32_MAX" >&5
 $as_echo "$ac_cv_have_decl_UINT32_MAX" >&6; }
-if test $ac_cv_have_decl_UINT32_MAX = yes; then
+if test "x$ac_cv_have_decl_UINT32_MAX" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_UINT32_MAX 1
@@ -27941,7 +27948,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_EFTYPE" >&5
 $as_echo "$ac_cv_have_decl_EFTYPE" >&6; }
-if test $ac_cv_have_decl_EFTYPE = yes; then
+if test "x$ac_cv_have_decl_EFTYPE" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_EFTYPE 1
@@ -28003,7 +28010,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_EILSEQ" >&5
 $as_echo "$ac_cv_have_decl_EILSEQ" >&6; }
-if test $ac_cv_have_decl_EILSEQ = yes; then
+if test "x$ac_cv_have_decl_EILSEQ" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_EILSEQ 1
@@ -28105,7 +28112,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_wchar_t" >&5
 $as_echo "$ac_cv_type_wchar_t" >&6; }
-if test $ac_cv_type_wchar_t = yes; then
+if test "x$ac_cv_type_wchar_t" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_WCHAR_T 1
@@ -28361,11 +28368,13 @@ done
 case $ac_lo in
 ?*) ac_cv_sizeof_wchar_t=$ac_lo;;
 '') if test "$ac_cv_type_wchar_t" = yes; then
-     { { $as_echo "$as_me:$LINENO: error: cannot compute sizeof (wchar_t)
+     { { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: cannot compute sizeof (wchar_t)
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: cannot compute sizeof (wchar_t)
 See \`config.log' for more details." >&2;}
-   { (exit 77); exit 77; }; }
+   { (exit 77); exit 77; }; }; }
    else
      ac_cv_sizeof_wchar_t=0
    fi ;;
@@ -28441,11 +28450,13 @@ sed 's/^/| /' conftest.$ac_ext >&5
 
 ( exit $ac_status )
 if test "$ac_cv_type_wchar_t" = yes; then
-     { { $as_echo "$as_me:$LINENO: error: cannot compute sizeof (wchar_t)
+     { { $as_echo "$as_me:$LINENO: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+{ { $as_echo "$as_me:$LINENO: error: cannot compute sizeof (wchar_t)
 See \`config.log' for more details." >&5
 $as_echo "$as_me: error: cannot compute sizeof (wchar_t)
 See \`config.log' for more details." >&2;}
-   { (exit 77); exit 77; }; }
+   { (exit 77); exit 77; }; }; }
    else
      ac_cv_sizeof_wchar_t=0
    fi
@@ -28772,7 +28783,7 @@ fi
 $as_echo "$ac_cv_header_sys_mkdev_h" >&6; }
 
 fi
-if test $ac_cv_header_sys_mkdev_h = yes; then
+if test "x$ac_cv_header_sys_mkdev_h" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define MAJOR_IN_MKDEV 1
@@ -28915,7 +28926,7 @@ fi
 $as_echo "$ac_cv_header_sys_sysmacros_h" >&6; }
 
 fi
-if test $ac_cv_header_sys_sysmacros_h = yes; then
+if test "x$ac_cv_header_sys_sysmacros_h" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define MAJOR_IN_SYSMACROS 1
@@ -29452,7 +29463,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_strerror_r" >&5
 $as_echo "$ac_cv_have_decl_strerror_r" >&6; }
-if test $ac_cv_have_decl_strerror_r = yes; then
+if test "x$ac_cv_have_decl_strerror_r" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_STRERROR_R 1
@@ -29559,8 +29570,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -29789,8 +29801,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -29862,7 +29875,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_intl_strftime" >&5
 $as_echo "$ac_cv_lib_intl_strftime" >&6; }
-if test $ac_cv_lib_intl_strftime = yes; then
+if test "x$ac_cv_lib_intl_strftime" = x""yes; then
   cat >>confdefs.h <<\_ACEOF
 #define HAVE_STRFTIME 1
 _ACEOF
@@ -29964,8 +29977,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30055,7 +30069,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_func__doprnt" >&5
 $as_echo "$ac_cv_func__doprnt" >&6; }
-if test $ac_cv_func__doprnt = yes; then
+if test "x$ac_cv_func__doprnt" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_DOPRNT 1
@@ -30160,8 +30174,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30265,8 +30280,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30369,8 +30385,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30471,8 +30488,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30576,8 +30594,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30682,8 +30701,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30786,8 +30806,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30890,8 +30911,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -30995,8 +31017,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -31065,7 +31088,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_D_MD_ORDER" >&5
 $as_echo "$ac_cv_have_decl_D_MD_ORDER" >&6; }
-if test $ac_cv_have_decl_D_MD_ORDER = yes; then
+if test "x$ac_cv_have_decl_D_MD_ORDER" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_D_MD_ORDER 1
@@ -31221,7 +31244,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_FNM_LEADING_DIR" >&5
 $as_echo "$ac_cv_have_decl_FNM_LEADING_DIR" >&6; }
-if test $ac_cv_have_decl_FNM_LEADING_DIR = yes; then
+if test "x$ac_cv_have_decl_FNM_LEADING_DIR" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_FNM_LEADING_DIR 1
@@ -31486,8 +31509,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -31636,8 +31660,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -31712,7 +31737,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_attr_setxattr" >&5
 $as_echo "$ac_cv_lib_attr_setxattr" >&6; }
-if test $ac_cv_lib_attr_setxattr = yes; then
+if test "x$ac_cv_lib_attr_setxattr" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBATTR 1
 _ACEOF
@@ -31813,8 +31838,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -31914,8 +31940,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -32015,8 +32042,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -32118,8 +32146,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -32283,8 +32312,9 @@ ac_res=`eval 'as_val=${'$as_ac_Header'}
 $as_echo "$ac_res" >&6; }
 
 fi
-if test `eval 'as_val=${'$as_ac_Header'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_Header'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
@@ -32359,7 +32389,7 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_acl_acl_get_file" >&5
 $as_echo "$ac_cv_lib_acl_acl_get_file" >&6; }
-if test $ac_cv_lib_acl_acl_get_file = yes; then
+if test "x$ac_cv_lib_acl_acl_get_file" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBACL 1
 _ACEOF
@@ -32463,8 +32493,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -32581,7 +32612,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_type_acl_permset_t" >&5
 $as_echo "$ac_cv_type_acl_permset_t" >&6; }
-if test $ac_cv_type_acl_permset_t = yes; then
+if test "x$ac_cv_type_acl_permset_t" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_ACL_PERMSET_T 1
@@ -32688,8 +32719,9 @@ ac_res=`eval 'as_val=${'$as_ac_var'}
 		 $as_echo "$as_val"'`
 	       { $as_echo "$as_me:$LINENO: result: $ac_res" >&5
 $as_echo "$ac_res" >&6; }
-if test `eval 'as_val=${'$as_ac_var'}
-		 $as_echo "$as_val"'` = yes; then
+as_val=`eval 'as_val=${'$as_ac_var'}
+		 $as_echo "$as_val"'`
+   if test "x$as_val" = x""yes; then
   cat >>confdefs.h <<_ACEOF
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
@@ -32754,7 +32786,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_ACL_USER" >&5
 $as_echo "$ac_cv_have_decl_ACL_USER" >&6; }
-if test $ac_cv_have_decl_ACL_USER = yes; then
+if test "x$ac_cv_have_decl_ACL_USER" = x""yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_ACL_USER 1
@@ -33176,7 +33208,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_optarg" >&5
 $as_echo "$ac_cv_have_decl_optarg" >&6; }
-if test $ac_cv_have_decl_optarg = yes; then
+if test "x$ac_cv_have_decl_optarg" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_OPTARG 1
@@ -33245,7 +33277,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:$LINENO: result: $ac_cv_have_decl_optind" >&5
 $as_echo "$ac_cv_have_decl_optind" >&6; }
-if test $ac_cv_have_decl_optind = yes; then
+if test "x$ac_cv_have_decl_optind" = x""yes; then
 
 cat >>confdefs.h <<_ACEOF
 #define HAVE_DECL_OPTIND 1
@@ -33289,8 +33321,8 @@ _ACEOF
     case $ac_val in #(
     *${as_nl}*)
       case $ac_var in #(
-      *_cv_*) { $as_echo "$as_me:$LINENO: WARNING: Cache variable $ac_var contains a newline." >&5
-$as_echo "$as_me: WARNING: Cache variable $ac_var contains a newline." >&2;} ;;
+      *_cv_*) { $as_echo "$as_me:$LINENO: WARNING: cache variable $ac_var contains a newline" >&5
+$as_echo "$as_me: WARNING: cache variable $ac_var contains a newline" >&2;} ;;
       esac
       case $ac_var in #(
       _ | IFS | as_nl) ;; #(
@@ -33745,7 +33777,7 @@ exec 6>&1
 # values after options handling.
 ac_log="
 This file was extended by libarchive $as_me 2.7.0, which was
-generated by GNU Autoconf 2.62.  Invocation command line was
+generated by GNU Autoconf 2.63.  Invocation command line was
 
   CONFIG_FILES    = $CONFIG_FILES
   CONFIG_HEADERS  = $CONFIG_HEADERS
@@ -33758,6 +33790,15 @@ on `(hostname || uname -n) 2>/dev/null |
 
 _ACEOF
 
+case $ac_config_files in *"
+"*) set x $ac_config_files; shift; ac_config_files=$*;;
+esac
+
+case $ac_config_headers in *"
+"*) set x $ac_config_headers; shift; ac_config_headers=$*;;
+esac
+
+
 cat >>$CONFIG_STATUS <<_ACEOF || ac_write_fail=1
 # Files that config.status was made for.
 config_files="$ac_config_files"
@@ -33771,11 +33812,12 @@ ac_cs_usage="\
 \`$as_me' instantiates files from templates according to the
 current configuration.
 
-Usage: $0 [OPTIONS] [FILE]...
+Usage: $0 [OPTION]... [FILE]...
 
   -h, --help       print this help, then exit
   -V, --version    print version number and configuration settings, then exit
-  -q, --quiet      do not print progress messages
+  -q, --quiet, --silent
+                   do not print progress messages
   -d, --debug      don't remove temporary files
       --recheck    update $as_me by reconfiguring in the same conditions
   --file=FILE[:TEMPLATE]
@@ -33798,7 +33840,7 @@ _ACEOF
 cat >>$CONFIG_STATUS <<_ACEOF || ac_write_fail=1
 ac_cs_version="\\
 libarchive config.status 2.7.0
-configured by $0, generated by GNU Autoconf 2.62,
+configured by $0, generated by GNU Autoconf 2.63,
   with options \\"`$as_echo "$ac_configure_args" | sed 's/^ //; s/[\\""\`\$]/\\\\&/g'`\\"
 
 Copyright (C) 2008 Free Software Foundation, Inc.
@@ -34004,7 +34046,8 @@ for ac_last_try in false false false fal
 $as_echo "$as_me: error: could not make $CONFIG_STATUS" >&2;}
    { (exit 1); exit 1; }; }
 
-  if test `sed -n "s/.*$ac_delim\$/X/p" conf$$subs.awk | grep -c X` = $ac_delim_num; then
+  ac_delim_n=`sed -n "s/.*$ac_delim\$/X/p" conf$$subs.awk | grep -c X`
+  if test $ac_delim_n = $ac_delim_num; then
     break
   elif $ac_last_try; then
     { { $as_echo "$as_me:$LINENO: error: could not make $CONFIG_STATUS" >&5
@@ -34209,9 +34252,9 @@ cat >>$CONFIG_STATUS <<_ACEOF || ac_writ
   }
   split(mac1, mac2, "(") #)
   macro = mac2[1]
+  prefix = substr(line, 1, index(line, defundef) - 1)
   if (D_is_set[macro]) {
     # Preserve the white space surrounding the "#".
-    prefix = substr(line, 1, index(line, defundef) - 1)
     print prefix "define", macro P[macro] D[macro]
     next
   } else {
@@ -34219,7 +34262,7 @@ cat >>$CONFIG_STATUS <<_ACEOF || ac_writ
     # in the case of _POSIX_SOURCE, which is predefined and required
     # on some systems where configure will not decide to define it.
     if (defundef == "undef") {
-      print "/*", line, "*/"
+      print "/*", prefix defundef, macro, "*/"
       next
     }
   }
@@ -34243,8 +34286,8 @@ do
   esac
   case $ac_mode$ac_tag in
   :[FHL]*:*);;
-  :L* | :C*:*) { { $as_echo "$as_me:$LINENO: error: Invalid tag $ac_tag." >&5
-$as_echo "$as_me: error: Invalid tag $ac_tag." >&2;}
+  :L* | :C*:*) { { $as_echo "$as_me:$LINENO: error: invalid tag $ac_tag" >&5
+$as_echo "$as_me: error: invalid tag $ac_tag" >&2;}
    { (exit 1); exit 1; }; };;
   :[FH]-) ac_tag=-:-;;
   :[FH]*) ac_tag=$ac_tag:$ac_tag.in;;
@@ -34737,7 +34780,7 @@ if test "$no_create" != yes; then
   $ac_cs_success || { (exit 1); exit 1; }
 fi
 if test -n "$ac_unrecognized_opts" && test "$enable_option_checking" != no; then
-  { $as_echo "$as_me:$LINENO: WARNING: Unrecognized options: $ac_unrecognized_opts" >&5
-$as_echo "$as_me: WARNING: Unrecognized options: $ac_unrecognized_opts" >&2;}
+  { $as_echo "$as_me:$LINENO: WARNING: unrecognized options: $ac_unrecognized_opts" >&5
+$as_echo "$as_me: WARNING: unrecognized options: $ac_unrecognized_opts" >&2;}
 fi
 
